store/items.ts

```diff
@@ -0,0 +1,71 @@
+import Vue from 'vue';
+import { BaseItemDto } from '@jellyfin/client-axios';
+import { ActionTree, GetterTree, MutationTree } from 'vuex';
+import map from 'lodash/map';
+import some from 'lodash/some';
+import forEach from 'lodash/forEach';
```

Comment:
@Codex- See here https://github.com/jellyfin/jellyfin-vue/pull/725

Reply:
@Codex- Oh, I understand now. I thought you were mentioning the import thing because you raised the topic in another comment as well.

See here: 

In general, all the lodash functions seems to perform better than the native counterparts.

Reply:
Lodash is faster for me (Latest Firefox on Linux):
![image](https://user-images.githubusercontent.com/19396809/110228854-d3815f80-7f04-11eb-8f75-6c2971a8399c.png)

Same thing on the latest Chromium, where the difference is even bigger:
![image](https://user-images.githubusercontent.com/19396809/110228877-0592c180-7f05-11eb-8f48-92335bd721cd.png)

Reply:
> I really don't think it's worth it unless you're explicitly targeting ancient versions of spidermonkey / v8.

Well, we're technically planning support for old versions of WebOS and Tizen, so Lodash makes sense there. You're also (afaik) the only one between the four of us who got better scores on native than lodash for these benchmarks, so I'm not sure what's up.

The TV UI is pretty far off, but the performance difference, from what we've seen, is pretty negligible (or in favor of Lodash, though not in your case) on evergreen browsers, while older ones will benefit from Lodash, so we may as well use it.

Reply:
Sorry to join in here, but I too got better/equal *native scores* (linux-arch, ff & brave).
However, this only proves the point that its incredibly system, OS and environment dependent and with the target of aincient platforms such as WebOS or Tizen (Browser support speaking) there likely is no way around *lodash*.

(Disclaimer: I am no Web dev, so my knowledge is very limited in this area)